test/conformance/cts_exe.py

```diff
@@ -18,18 +20,24 @@
 
     parser = ArgumentParser()
     parser.add_argument("--test_command", help="Ctest test case")
-
     args = parser.parse_args()
-    result = subprocess.Popen([args.test_command, '--gtest_brief=1'], stdout = subprocess.PIPE, text = True)  # nosec B603
+    result = subprocess.Popen([args.test_command, '--gtest_brief=1'], stdout=subprocess.PIPE,
+                              stderr=subprocess.STDOUT, text=True)  # nosec B603
 
     pat = re.compile(r'\[( )*FAILED( )*\]')
-    for line in result.stdout.readlines():
+    output_list = []
```

Comment:
Is it really needed? Can't you just print `result.stdout` later instead of creating a new list for the same content?

Reply:
yes it is needed. I can't print/use result.stdout again after the loop.